scripts/plugins/storage/sqlite/init.sql

```diff
@@ -0,0 +1,971 @@
+    asset_code        character varying(50)     NOT NULL,
+    CONSTRAINT omf_created_objects_pkey PRIMARY KEY (configuration_key,type_id, asset_code),
+    CONSTRAINT omf_created_objects_fk1 FOREIGN KEY (configuration_key)
+    REFERENCES configuration (key) MATCH SIMPLE
+            ON UPDATE NO ACTION
+            ON DELETE NO ACTION );
+
+
+-- Backups information
+-- Stores information about executed backups
+CREATE TABLE foglamp.backups (
+    id         INTEGER                 PRIMARY KEY AUTOINCREMENT,
+    file_name  character varying(255)  NOT NULL DEFAULT '',                   -- Backup file name, expressed as absolute path
+    ts         DATETIME DEFAULT (STRFTIME('%Y-%m-%d %H:%M:%f+00:00', 'NOW')), -- Backup creation timestamp
+    type       integer                 NOT NULL,                              -- Backup type : 1-Full, 2-Incremental
+    status     integer                 NOT NULL,                              -- Backup status :
+                                                                              --   1-Running
+                                                                              --   2-Completed
+                                                                              --   3-Cancelled
+                                                                              --   4-Interrupted
+                                                                              --   5-Failed
+                                                                              --   6-Restored backup
+    exit_code  integer );                                                     -- Process exit status code
+
+
+-- FogLAMP DB version: keeps the schema version id
+CREATE TABLE foglamp.version (id CHAR(10));
+
+
+----------------------------------------------------------------------
+-- Initialization phase - DML
+----------------------------------------------------------------------
+
+-- Roles
+DELETE FROM foglamp.roles;
+INSERT INTO foglamp.roles ( name, description )
+     VALUES ('admin', 'for the users having all CRUD privileges including other admin users'),
+            ('user', 'all CRUD operations and self profile management');
+
+-- Users
+DELETE FROM foglamp.users;
+INSERT INTO foglamp.users ( uname, pwd, role_id, description )
+     VALUES ('admin', '3a86096e7a7c123ba0bc3dfb7a1d350541649f1ff1aff1f37e0dc1ee4175b112:3759bf3302f5481e8c9cc9472c6088ac', 1, 'admin user'),
+            ('user', '3a86096e7a7c123ba0bc3dfb7a1d350541649f1ff1aff1f37e0dc1ee4175b112:3759bf3302f5481e8c9cc9472c6088ac', 2, 'normal user');
+
+-- User password history
+DELETE FROM foglamp.user_pwd_history;
+INSERT INTO foglamp.user_pwd_history ( user_id, pwd )
+    VALUES (1, '3a86096e7a7c123ba0bc3dfb7a1d350541649f1ff1aff1f37e0dc1ee4175b112:3759bf3302f5481e8c9cc9472c6088ac'),
+           (2, '3a86096e7a7c123ba0bc3dfb7a1d350541649f1ff1aff1f37e0dc1ee4175b112:3759bf3302f5481e8c9cc9472c6088ac');
```

Comment:
these entries are not needed

Reply:
@praveen-garg those are present in postgres/init.sql as well

Reply:
https://github.com/foglamp/FogLAMP/blob/develop/scripts/plugins/storage/postgres/init.sql#L766

Reply:
my bad, fixed